docs-src/how-to-create-layoutbuilder.md

```diff
@@ -14,10 +14,303 @@ kernelspec:
 How to create arrays with LayoutBuilder (more control)
 ======================================================
 
-**This is a stub:** I intend to write this article, but haven't yet.
+What is LayoutBuilder
+---------------------
 
-If you need it soon, create an issue saying so and I'll make it a higher priority.
+`ak.layout.LayoutBuilder` is the low-level LayoutBuilder that builds layouts, or `ak.layout.Content` arrays. It must be initialized by a JSON string that represents a valid `ak.forms.Form`.
 
-[![](img/github-issues-documentation.png)](https://github.com/scikit-hep/awkward-1.0/issues/new?assignees=&labels=docs&template=documentation.md&title=)
+A layout is the composable `ak.layout.Content` elements that determine how an array is structured. The layout may be considered a "low-level" view, as it distinguishes between arrays that have the same logical meaning (i.e. same JSON output and high-level `type`) but different
```

Comment:
Maybe clearer to say
```suggestion
A layout is build from composable `ak.layout.Content` elements that determine how an array is structured. The layout may be considered a "low-level" view, as it distinguishes between arrays that have the same logical meaning (i.e. same JSON output and high-level `type`) but different
```

Reply:
I think "is" is correct: a layout is the tree of nested `ak.layout.Content` nodes. To say that it's built from them implies that there's another step, analogous to the step from a Builder (ArrayBuilder or LayoutBuilder) to an array through `snapshot`.

"Layout" may simply be a synonym for `Content`, though it may be distinguished in that "layout" often refers to the whole tree and a `Content` object is specifically one node, since it has a meaning in code. Maybe that's what you meant by "built from."

In that case, how about "A layout consists of composable `ak.layout.Content`..."? The word "consists" is describing a static thing, whereas "built from" could describe a static thing or could describe a process like the one involving `snapshot`.